--- ../vendor/phpoffice/phpword/src/PhpWord/Writer/Word2007/Style/Paragraph.php	2017-11-23 22:31:22.993831901 -0300
+++ Paragraph.php	2017-11-23 22:48:57.811314887 -0300
@@ -100,7 +100,8 @@
 
         // Paragraph alignment
         if ('' !== $styles['alignment']) {
-        	$paragraphAlignment = new ParagraphAlignment($styles['alignment']);
+        	$styleConverted=(($styles['alignment']=='justify')?Jc::BOTH:$styles['alignment']);
+        	$paragraphAlignment = new ParagraphAlignment($styleConverted);//($styles['alignment']);
             $xmlWriter->startElement($paragraphAlignment->getName());
             foreach ($paragraphAlignment->getAttributes() as $attributeName => $attributeValue) {
                 $xmlWriter->writeAttribute($attributeName, $attributeValue);
